dev-lang/ocaml/ocaml-4.09.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+HOMEPAGE="https://ocaml.org/"
+SRC_URI="https://github.com/ocaml/ocaml/archive/${PV}.tar.gz -> ${P}.tar.gz"
+DESCRIPTION="Programming language supporting functional, imperative & object-oriented styles"
+
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="emacs flambda latex spacetime xemacs"
```

Comment:
is there a reason why ocamlopt was dismissed? It seems to be present in the old ebuild, and not much have changed since, they still provide it.

Hint:  the target is called `world.opt` now, so might be nice to look into it.

Reply:
I am actually calling `emake world.opt` already, as you can see. 😁 That's also the reason why I could drop `ocamlopt`. The world.opt target in version 4.09.0 works with parallel builds, and the USE flag in the 4.05 ebuild forced sequential builds.

Reply:
@rseichter, err, I had different idea in mind, you should call `world.opt` only on ocalmopt usage, and `world` otherwise, afair it gonna work properly.

Reply:
@Zlogene With respect to the [upstream installation description](http://caml.inria.fr/pub/distrib/ocaml-4.09/notes/INSTALL.adoc), I got the impression that it is preferable to always use the native-code compiler if that is possible. Given that the build is now keyworded ~amd64 only, on which the native-code compiler works, I thought it fitting to remove the USE flag.

However, if you really do want to keep `ocamlopt`, I can fit it back in, although I personally don't see an advantage in doing so at this time.

Reply:
@rseichter well, as you can see you broke the whole ml stuff by not introducing ocamlopt :)

Reply:
A fair point. 😉 I have put the USE flag back in.